lldb/test/API/functionalities/thread/finish-from-empty-func/main.c

```diff
@@ -2,6 +2,7 @@
 void done() {}
```

Comment:
Maybe this is a bit much, but you could add some statements in here like:
```
volatile int i = 1;
```
This way, if someone breaks your logic in the Python, it will fail on all platforms.